### DISCUSSION TITLE
About the kidney

### DISCUSSION THREAD

**Discussion by Carrie-nothing** [Open-Systems-Pharmacology/Forum#1049] [Questions & Problems] [answered]
Hello team,

I can choose to predict the blood concentration of the kidney  

But how can I get the blood concentration in the epithelial cells of the renal tubules?  

Best ,
Carrie.

**Comment by AndreDlm** [comment, marked as answer]
Hi @Carrie-nothing, currently the kidney compartment consists only of 4 compartments (plasma, blood cells, interstitial and intracellular - in case of the model structure for small molecules), but is doesn't explicitly contain a compartment for the epithelial cells of the renal tubules.

André

**Reply by Carrie-nothing** [reply]
thank you very much！
